### ISSUE TITLE
regression: conflicting implementations for From<&_>

### ISSUE THREAD

**Issue by Mark-Simulacrum** [rust-lang/rust#61474]
root: ocl - 1 detected crates which regressed due to this; cc @c0gent
<details>

```
 * ikachko/render_hash_2d: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/ikachko.render_hash_2d/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/ikachko.render_hash_2d/log.txt); cc @ikachko
```

</details>

root: rusqlite - 61 detected crates which regressed due to this; cc @jgallagher, @gwenn
<details>

```
 * citadel-crud-0.1.1: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/citadel-crud-0.1.1/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/citadel-crud-0.1.1/log.txt); cc @natepisarski
 * database_exporter-0.1.4: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/database_exporter-0.1.4/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/database_exporter-0.1.4/log.txt); cc @amilajack
 * flo_anim_sqlite-0.1.0: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/flo_anim_sqlite-0.1.0/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/flo_anim_sqlite-0.1.0/log.txt); cc @Logicalshift
 * flow_between-0.1.0: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/flow_between-0.1.0/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/flow_between-0.1.0/log.txt); cc @Logicalshift
 * generate-nix-pkg-0.3.0: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/generate-nix-pkg-0.3.0/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/generate-nix-pkg-0.3.0/log.txt); cc @P-E-Meunier
 * iot_center-0.0.1: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/iot_center-0.0.1/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/iot_center-0.0.1/log.txt); cc @babydragon
 * iron_rusqlite_middleware-0.1.2: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/iron_rusqlite_middleware-0.1.2/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/iron_rusqlite_middleware-0.1.2/log.txt); cc @KevinMidkiff
 * nickel_sqlite-0.4.0: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/nickel_sqlite-0.4.0/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/nickel_sqlite-0.4.0/log.txt); cc @flosse
 * rtq-0.1.0: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/rtq-0.1.0/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/rtq-0.1.0/log.txt); cc @funzoo
 * torus-0.1.1: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/reg/torus-0.1.1/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/reg/torus-0.1.1/log.txt); cc @MarkJr94
 * ChrisMacNaughton/cargo-cacher: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/ChrisMacNaughton.cargo-cacher/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/ChrisMacNaughton.cargo-cacher/log.txt); cc @ChrisMacNaughton
 * Ophirr33/NEUCourseNotification: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/Ophirr33.NEUCourseNotification/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/Ophirr33.NEUCourseNotification/log.txt); cc @Ophirr33
 * Vzaa/contacts2dbtovcf: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/Vzaa.contacts2dbtovcf/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/Vzaa.contacts2dbtovcf/log.txt); cc @Vzaa
 * cholcombe973/dinner: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/cholcombe973.dinner/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/cholcombe973.dinner/log.txt); cc @cholcombe973
 * cholcombe973/gluster-charm: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/cholcombe973.gluster-charm/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/cholcombe973.gluster-charm/log.txt); cc @cholcombe973
 * cholcombe973/yesterday_weather: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/cholcombe973.yesterday_weather/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/cholcombe973.yesterday_weather/log.txt); cc @cholcombe973
 * cristicbz/uok-server: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/cristicbz.uok-server/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/cristicbz.uok-server/log.txt); cc @cristicbz
 * danielsakhapov/asterius_chat_server: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/danielsakhapov.asterius_chat_server/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/danielsakhapov.asterius_chat_server/log.txt); cc @danielsakhapov
 * darakus188/red-shale: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/darakus188.red-shale/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/darakus188.red-shale/log.txt); cc @darakus188
 * denysvitali/LaunchBot: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/denysvitali.LaunchBot/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/denysvitali.LaunchBot/log.txt); cc @denysvitali
 * denysvitali/TheRealLPTBot: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/denysvitali.TheRealLPTBot/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/denysvitali.TheRealLPTBot/log.txt); cc @denysvitali
 * diract/droxy: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/diract.droxy/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/diract.droxy/log.txt); cc @diract
 * drozdziak1/rust-junk: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/drozdziak1.rust-junk/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/drozdziak1.rust-junk/log.txt); cc @drozdziak1
 * erikjohnston/matrix-media-store-size: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/erikjohnston.matrix-media-store-size/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/erikjohnston.matrix-media-store-size/log.txt); cc @erikjohnston
 * esnowkropla/iron_test: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/esnowkropla.iron_test/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/esnowkropla.iron_test/log.txt); cc @esnowkropla
 * funzoo/rtq: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/funzoo.rtq/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/funzoo.rtq/log.txt); cc @funzoo
 * gsquire/yaus: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/gsquire.yaus/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/gsquire.yaus/log.txt); cc @gsquire
 * hagt/semnet-web: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/hagt.semnet-web/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/hagt.semnet-web/log.txt); cc @hagt
 * hekrause/garmin_translate: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/hekrause.garmin_translate/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/hekrause.garmin_translate/log.txt); cc @hekrause
 * jchristgit/rusday: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/jchristgit.rusday/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/jchristgit.rusday/log.txt); cc @jchristgit
 * jelford/squirrel: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/jelford.squirrel/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/jelford.squirrel/log.txt); cc @jelford
 * justinnoah/rtracker: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/justinnoah.rtracker/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/justinnoah.rtracker/log.txt); cc @justinnoah
 * keringar/dci-bot: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/keringar.dci-bot/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/keringar.dci-bot/log.txt); cc @keringar
 * kohnakagawa/SQLNetworkServer: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/kohnakagawa.SQLNetworkServer/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/kohnakagawa.SQLNetworkServer/log.txt); cc @kohnakagawa
 * loopmachine/slack-archive: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/loopmachine.slack-archive/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/loopmachine.slack-archive/log.txt); cc @loopmachine
 * louwie17/sample-api: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/louwie17.sample-api/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/louwie17.sample-api/log.txt); cc @louwie17
 * maufl/gb2idb: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/maufl.gb2idb/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/maufl.gb2idb/log.txt); cc @maufl
 * mhsjlw/nod: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/mhsjlw.nod/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/mhsjlw.nod/log.txt); cc @mhsjlw
 * mitchelhaan/cpanel-dyndns: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/mitchelhaan.cpanel-dyndns/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/mitchelhaan.cpanel-dyndns/log.txt); cc @mitchelhaan
 * murnux/RuhRoh: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/murnux.RuhRoh/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/murnux.RuhRoh/log.txt); cc @murnux
 * murnux/roffee: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/murnux.roffee/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/murnux.roffee/log.txt); cc @murnux
 * nick-benoit14/inventory: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/nick-benoit14.inventory/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/nick-benoit14.inventory/log.txt); cc @nick-benoit14
 * nullobject/eventlogd: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/nullobject.eventlogd/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/nullobject.eventlogd/log.txt); cc @nullobject
 * rharriso/jot: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/rharriso.jot/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/rharriso.jot/log.txt); cc @rharriso
 * roller/bazbot: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/roller.bazbot/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/roller.bazbot/log.txt); cc @roller
 * rskew/autonomic-rust: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/rskew.autonomic-rust/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/rskew.autonomic-rust/log.txt); cc @rskew
 * ryanpbrewster/task-queue: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/ryanpbrewster.task-queue/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/ryanpbrewster.task-queue/log.txt); cc @ryanpbrewster
 * samcrow/netcheck: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/samcrow.netcheck/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/samcrow.netcheck/log.txt); cc @samcrow
 * sbwtw/tower: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/sbwtw.tower/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/sbwtw.tower/log.txt); cc @sbwtw
 * silago/foxnotdead: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/silago.foxnotdead/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/silago.foxnotdead/log.txt); cc @silago
 * simias/cassettes: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/simias.cassettes/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/simias.cassettes/log.txt); cc @simias
 * stan4cb/rs-gold-report: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/stan4cb.rs-gold-report/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/stan4cb.rs-gold-report/log.txt); cc @stan4cb
 * synac-chat/client: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/synac-chat.client/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/synac-chat.client/log.txt); cc @synac-chat
 * synac-chat/client-gtk: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/synac-chat.client-gtk/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/synac-chat.client-gtk/log.txt); cc @synac-chat
 * thomcc/anonymize-places: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/thomcc.anonymize-places/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/thomcc.anonymize-places/log.txt); cc @thomcc
 * tollyx/plankboat-rs: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/tollyx.plankboat-rs/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/tollyx.plankboat-rs/log.txt); cc @tollyx
 * wolfmankurd/jukebox-rust: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/wolfmankurd.jukebox-rust/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/wolfmankurd.jukebox-rust/log.txt); cc @wolfmankurd
 * wyvie/stallman-bot: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/wyvie.stallman-bot/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/wyvie.stallman-bot/log.txt); cc @wyvie
 * xzfc/holy-bot-nu-serv: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/xzfc.holy-bot-nu-serv/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/xzfc.holy-bot-nu-serv/log.txt); cc @xzfc
 * yeliknewo/wikiracer: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/yeliknewo.wikiracer/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/yeliknewo.wikiracer/log.txt); cc @yeliknewo
 * yotarok/weblibri: [start](https://crater-reports.s3.amazonaws.com/beta-1.36-2/1.35.0/gh/yotarok.weblibri/log.txt) v. [end](https://crater-reports.s3.amazonaws.com/beta-1.36-2/beta-2019-05-30/gh/yotarok.weblibri/log.txt); cc @yotarok
```

</details>

**Comment by Centril** [comment]
Also guessing this is due to https://github.com/rust-lang/rust/pull/49799.

**Comment by gwenn** [comment]
Regarding `rustyline`, this was fixed in version [0.14.0](https://github.com/jgallagher/rusqlite/issues/313), no?

**Comment by pnkfelix** [comment]
It does look like the `rusqlite` issue is addressed in a follow-up version.

However, the followup version was an increment of the minor version number on a `0.x` product, so uses of `cargo update` will not automatically inherit the fix.

(At the same time, I don't think there is a way to fix this via a patch release, since it is inherently removing a trait impl, right?)

Nonetheless, I think this is expected fallout of #49799. Closing as such.
